src/main/resources/application.yaml

```diff
@@ -1,34 +1,7 @@
 # Quarkus will choose application.yaml over application.properties.
 quarkus:
-  http:
-    port: 8080
-    read-timeout: 30m
-    limits:
-      max-body-size: 500M
   package:
     type: uber-jar
-  application:
-    name: "indy-tracking-service"
-  # Logging (disable console on prod)
```

Comment:
Same as above.

Reply:
I'd recommend to keep this as a general application name.

Reply:
added it back